viashpy/testing.py

```diff
@@ -184,13 +190,16 @@ def wrapper(*args, **kwargs):
     if viash_source_config_path.is_file():
 
         @run_and_handle_errors
-        def wrapper(args_as_list, platform: Platform = "docker"):
+        def wrapper(
+            args_as_list, engine: Engine | None = None, platform: Platform | None = None
```

Comment:
We can assume that `runner` is `executable`, right?

Reply:
I'm not sure what you are referring to. `runner` is not a variable in this script?

Reply:
Yes, I think currently we test the executable runner in almost all tests and hope that that guarantees that component works as intented when using other runners.  Implementing another runner could be done I, but I have the impression that this would not be a quick undertaking.